project-canvas/src/components/projects-view/projectsData.ts

```diff
@@ -0,0 +1,39 @@
+// import IconStar from "@tabler/icons"
+
+export const data = [
+  {
+    // Star: IconStar,
+    Name: "Project 1",
+    Key: "P1",
+    Type: "project1 Type",
+    Lead: "Ihsen",
+  },
+  {
+    // Star: IconStar,
+    Name: "Project 2",
+    Key: "P2",
+    Type: "project2 Type",
+    Lead: "Alex",
+  },
+  {
+    // Star: IconStar,
+    Name: "Project 3",
+    Key: "P3",
+    Type: "project3 Type",
+    Lead: "Chris",
+  },
+  {
+    // Star: IconStar,
+    Name: "Project 4",
+    Key: "P4",
+    Type: "project4 Type",
+    Lead: "Oussema",
+  },
+  {
+    // Star: IconStar,
+    Name: "Project 5",
+    Key: "P5",
+    Type: "project5 Type",
+    Lead: "Bic",
+  },
+]
```

Comment:
There is no need to this anymore as the data is being fetched from the providers.